test/androidTest/java/org/thoughtcrime/securesms/espresso/Helper.java

```diff
@@ -14,4 +14,49 @@ public Helper(ActivityTestRule<ConversationListActivity> activityRule) {
     public ConversationsHelper goConversations() {
         return new ConversationsHelper(new HelperSecret());
     }
+
+    public PreferencesHelper resetNickname() {
+        return new PreferencesHelper(new HelperSecret());
+    }
+
+    public PreferencesHelper setNickname(String message) {
```

Comment:
should also be in PreferencesHelper class. Helper is extended by helpers for all pages ...